examples/modules/modules.bal

```diff
@@ -1,6 +1,6 @@
 // This example imports a module. You can only refer to the public symbols of
 // an imported module.
-import ballerina/math;
+import ballerina/lang.'float;
```

Comment:
Not the best example, since there's a predeclared import for this. Need to find a suitable alternative with a constant.

Reply:
Created https://github.com/ballerina-platform/ballerina-distribution/issues/1247.